fetchai/ledger/api/__init__.py

```diff
@@ -44,6 +47,11 @@ def __init__(self, host, port):
         self.tx = TransactionApi(host, port)
         self.server = ServerApi(host, port)
 
+        # Check that ledger version is compatible with API version
+        server_version = self.server.version().lstrip('v')
+        if not all(semver.match(server_version, c) for c in __compatible__):
+            raise RuntimeError("Ledger version running on server is not compatible with this API")
```

Comment:
Minor: Can we add returned version number and compatibility in this error message

Reply:
Added in most recent commit.